spec/index.bs

```diff
@@ -1894,6 +1900,111 @@ dictionary GPUVertexAttributeDescriptor {
 };
 </script>
 
+## {{GPUPipelineBase/getBindGroupLayout()|getBindGroupLayout}} ## {#get-bind-group-layout}
+
+<dl dfn-type="method" dfn-for="GPUPipelineBase">
+    : <dfn>getBindGroupLayout(unsigned long index)</dfn>
+    ::
+        <div algorithm="GPUProgrammablePipeline.getBindGroupLayout(unsigned long index)">
+            1. If |index| is greater or equal to {{GPULimits/maxBindGroups}} :
```

Comment:
This could be covered by the condition that checks if the corresponding pipeline layout has the index. Is there a value in making this a separate check and error type?

Reply:
> This could be covered by the condition that checks if the corresponding pipeline layout has the index.

There isn't such a check in this algorithm, or in `GPUDevice.createBindGroupLayout` so I'm not sure what you mean.

This check throws a JS error because it helps statically allocate the correct number of slots for memoizing the `GPUBindGroupLayouts` in `GPUPipelineBase`. Also it can be useful to return "empty" bind group layouts for group indices that don't have any bindings in the shader.